src/clientfactory.ts

```diff
@@ -40,12 +46,11 @@ export class DiscordClientFactory {
         // We just need to make sure we have a bearer token.
         // Create a new Bot client.
         this.botClient = new DiscordClient({
-            fetchAllMembers: this.config.usePrivilegedIntents,
-            messageCacheLifetime: 5,
-            ws: {
-                intents: this.config.usePrivilegedIntents ? Intents.ALL : Intents.NON_PRIVILEGED,
-            },
-            shardCount: this.config.shards,
+            makeCache: Options.cacheWithLimits({
+                MessageManager: 12
```

Comment:
iirc there was no way for me to set a "lifetime" like before, but instead a number of messages in a channel until the message is removed from cache? I remember struggling to find how to use `cacheWithLimits`, and I still don't feel like I understand it...